README.md

```diff
@@ -1,18 +1,29 @@
-=== AMP ===
-Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo, johnwatkins0, joshuawold, ryankienstra
-Tags: amp, mobile, optimization, accelerated mobile pages, framework, components, blocks, performance, ux, seo, official
-Requires at least: 4.9
-Tested up to: 5.6
-Stable tag: 2.0.10
-License: GPLv2 or later
-License URI: http://www.gnu.org/licenses/gpl-2.0.html
-Requires PHP: 5.6
+# AMP
 
-The Official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile Pages, AMP enables great experiences across both mobile and desktop.
+Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo,   johnwatkins0, joshuawold, ryankienstra
```

Comment:
This line does not have the required 2 spaces at the end and so it does not render correctly:

![image](https://user-images.githubusercontent.com/16200219/105651746-95c5fb80-5eaf-11eb-9c60-5dee95b24636.png)

```suggestion
Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo,   johnwatkins0, joshuawold, ryankienstra  
```